pages/item/_itemId/index.vue

```diff
@@ -78,24 +78,39 @@ export default Vue.extend({
         return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Backdrop`;
       }
     },
+    getEndsAtTime(ticks: number): string {
+      const ms = ticks / 10000;
+      const endTimeLong = new Date(Date.now() + ms);
+      const endTimeShort = endTimeLong.toLocaleString('en-US', {
+        hour: 'numeric',
+        minute: 'numeric'
+      });
+
+      return `${this.$t('endsAt')}: ${endTimeShort}`;
```

Comment:
This should also use a parameter, like for season numbers.

I think having "sentences" (ie "Ends at 11:56pm") also looks better UI-wise than the colons, but that's debatable.

It could also likely be a property, I think, but I'm not too sure. Can always be refactored later.

Reply:
Sounds good to me. I've updated it